### PR TITLE
Add prettier config, styled-components; Lay out basic styling

### PR DESCRIPTION
**Pull request by ericliu121187**
https://trello.com/c/VX8Ol9xK/166-3-basic-styling

* [Demo](https://cl.ly/3O1G091O1B0G) -- did basic styling, nothing is set in stone, open to suggestions, and we'll iterate on this 
* Add prettier config so now it auto-formats on save if you have the following vscode user settings and [prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) installed
```javascript
{
  // Set the default
  "editor.formatOnSave": false,
  // Enable per-language
    "[javascript]": {
      "editor.formatOnSave": true
    }
}
```
* Add [styled-components](styled-components.com)

### CODE REVIEW

File: frontend/src/components/App.js [hunk 1]
```diff
@@ -1,30 +1,52 @@
...
-import { HandSubmission } from './HandSubmission';
 import LoginField from './LoginField';
 import PokerTable from './PokerTable';
 import React from 'react';
+import styled from 'styled-components';
 import { compose, withHandlers, withState } from 'recompose';
 
+const Header = styled.header`
+  font-size: ${({ isLoggedIn }) => (isLoggedIn ? '.625rem' : '1rem')};
+  text-align: ${({ isLoggedIn }) => (isLoggedIn ? 'left' : 'center')};
```
Comment (ericliu121187) [inline frontend/src/components/App.js]: Here we're using props in a styled component -- very cool

File: frontend/src/components/App.js [hunk 2]
```diff
@@ -1,30 +1,52 @@
...
   isLoggedIn,
   handleUsernameChange,
   handleLoginSubmit,
   username,
 }) {
   return (
-    <div>
-      <header>
+    <Wrapper>
+      <Header isLoggedIn={isLoggedIn}>
```
Comment (ericliu121187) [inline frontend/src/components/App.js]: Here we pass props to a styled component -- very cool

File: frontend/.prettierrc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "printWidth": 80,
+  "parser": "flow",
+  "singleQuote": true,
+  "trailingComma": "es5"
```
Comment (ericliu121187) [inline frontend/.prettierrc]: For a complete list of possible options -- http://json.schemastore.org/prettierrc

**Comment by txu89** [comment]
Need to review more before approving. Will do it tomorrow.

**Pull request merged**